src/arizona_template_compiler.erl

```diff
@@ -243,22 +255,42 @@ should_eval_expr(Expr, Macros) when map_size(Macros) > 0 ->
 should_eval_expr(_, _) ->
     false.
 
+expand_is_visible(true, _) ->
```

Comment:
It's better, even if ignored, that our `_` variables are named, so the code's easier to follow. It'd be `_State`, in this case. Not a strong rule, but a recommendation.

Reply:
This pattern occurs in several places. I'll remember this and name underscored variables in the next PRs.